content/en/docs/tasks/manage-kubernetes-objects/storage-version-migration.md

```diff
@@ -274,7 +274,7 @@ This migration can be achieved through _Storage Version Migration_ to migrate al
   kubectl apply -f migrate-crd.yaml
   ```
 - Monitor migration of secrets using status. Successful migration should have
-  _Succeeded_ condition set to _true_ in the status field. Get the migration resource
+  `Succeeded` condition set to "True" in the status field. Get the migration resource
```

Comment:
nit: we usually [don't](https://github.com/kubernetes/website/pull/45757#issuecomment-2032289358) put the string values of fields in code style

(I would omit the italics here)